docs/projects/load-project-yaml.ipynb

```diff
@@ -0,0 +1,1359 @@
+{
```

Comment:
<p>need to add remotely to <a href="https://app.reviewnb.com/mlrun/mlrun/pull/3640/#running-the-function-locally" target="_blank">run jobs locally</a>  &nbsp;and remotely</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3640/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>added</p>